src/main/scala/com/gd/twitteranalytics/JsonTranslator.scala

```diff
@@ -0,0 +1,23 @@
+package com.gd.twitteranalytics
+
+import org.apache.log4j.Logger
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+object JsonTranslator {
```

Comment:
the generic name implies that the object provides functionality for "working" with Json in general.
But in fact, a particular dataset gets processed.

E.g.:
`JsonTranslator#getFilteredDf` signature says about format and technology/framework, but not about actual business domain applicability. This is an anti-pattern in designing APIs. Please make it more specific, self-descriptive , reflecting **business** logic.